src/Task/Git/CommitMessage.php

```diff
@@ -96,7 +96,18 @@ public function run(ContextInterface $context)
         }
 
         if (count($exceptions)) {
-            return TaskResult::createFailed($this, $context, implode(PHP_EOL, $exceptions));
+            return TaskResult::createFailed(
+                $this,
+                $context,
+                implode(PHP_EOL, $exceptions) . <<<KAWAii
+</>
```

Comment:
The overuse of random colors also becomes hard to read. That is why the messages have specific colors. Green = ok, Red = not ok, Yellow = warning.
I think it's best to keep it consequent for now. If you do want the colors, you could still it to a custom formatter for your project to make it 100% Kawaii.